src/Common/Exceptions/DirectoryExistingException.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __DIRECTORY_EXISTING_EXCEPTION_H_
+#define __DIRECTORY_EXISTING_EXCEPTION_H_
+
+#include "BaseException.h"
+
+namespace FileManager {
+
+    class DirectoryExistingException : public BaseException {
+    public:
+
+        std::string path;
+
+        DirectoryExistingException(std::string path)
```

Comment:
What about?

```suggestion
        DirectoryAlreadyExistsException(std::string path)
```

Reply:
Thought about it, but I feel it's longer than needed.

Reply:
Not that much 😁